lib/Models/CswCatalogGroup.js

```diff
@@ -765,6 +768,27 @@ function createItemForUri(catalogGroup, record, uri, downloadUrls, legendUrl) {
   return catalogItem;
 }
 
+function getCatalogItemName(catalogGroup, record, uri) {
+  const properties = catalogGroup.itemProperties;
+
+  if (defined(properties)) {
+    const templateName = properties.itemNameTemplate;
+
+    if (defined(templateName)) {
+      if (typeof templateName === "string") {
+        return Mustache.render(templateName, { record, uri });
+      } else if (typeof templateName === "object") {
+        return Mustache.render(
+          templateName.template,
+          { record, uri },
```

Comment:
Unfortunately our documentation generation pipeline cannot deal with ES6/6+ syntax. Please replace both occurrences of { record, uri } with the old ES5 syntax: {record: record, uri: uri} (and then prettier will probably reformat that into multiple lines).

Reply:
Oh, I apologize, it wasn't clear to me, thanks for clarifying, I'll fix that

Reply:
Replaced that and also changed const to var